src/codeql.ts

```diff
@@ -256,40 +399,21 @@ export function getSarifOutputType(
 // Generates sarif from the given bqrs file, if query kind supports it
 async function generateSarif(
   codeql: string,
-  bqrs: string,
   nwo: string,
-  queryMetadata: QueryMetadata,
-  sarifOutputType: SarifOutputType,
   databaseName: string,
-  sourceLocationPrefix: string,
+  queryPackName: string,
   databaseSHA?: string,
 ): Promise<Sarif> {
-  const {
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars -- we are explicitly excluding this since it's calculated separately
-    kind,
-    id: queryId,
-    ...passthroughQueryMetadata
-  } = queryMetadata;
-
   const sarifFile = path.join("results", "results.sarif");
   await exec(codeql, [
-    "bqrs",
-    "interpret",
+    "database",
+    "interpret-results",
     "--format=sarif-latest",
     `--output=${sarifFile}`,
-    `-t=kind=${sarifOutputType}`,
-    `-t=id=${queryId || "remote-query"}`,
-    // Forward all of the query metadata.
-    ...Object.entries(passthroughQueryMetadata).map(
-      ([key, value]) => `-t=${key}=${value}`,
-    ),
     "--sarif-add-snippets",
     "--no-group-results",
-    // Hard-coded the source archive as src.zip inside the database, since that's
-    // where the CLI puts it. If this changes, we need to update this path.
-    `--source-archive=${databaseName}/src.zip`,
-    `--source-location-prefix=${sourceLocationPrefix}`,
-    bqrs,
+    databaseName,
+    queryPackName,
```

Comment:
Why are we using query pack name here rather than the query pack path? We seem to be using this all over the place, and sometimes (e.g. in `getRemoteQueryPackQueries`) we path both.

This isn't really being changed by this PR since the `getQueryPackName` function isn't being changed and the value was already being used in multiple places, but still it could be worth investigating what this value is and do we want to be using it over the path?

Reply:
Shall we deal with this as a follow up? I don't really know why we use pack name in some places and pack path in others.

Reply:
Created internal issue to cover this.